pkg/resourceinterpreter/configurableinterpreter/luavm/kube.go

```diff
@@ -5,6 +5,7 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/resource"
 
+	"github.com/karmada-io/karmada/pkg/resourceinterpreter/defaultinterpreter"
```

Comment:
It's easy to cause `import cycle` like this, so I suggest moving the functions to another place.